body.tex

```diff
@@ -0,0 +1,442 @@
+
+
+\newpage
+\section{Following activities} \label{sec:follows}
+
+The changes proposed in the previous section \ref{sec:proposed} can be implemented with very minimum impact to the day by day activities.
+
+The new functionalities introduced, can facilitate already the management of the conda environment as it is done now, and the development activities in general.
+
+However, the resolution of the problems exposed in \ref{sec:problems}, requires the reduction of the number of packages included in the shared stack, moving third party libraries in the conda environment.
+
+
+\subsection{Conda Packaging 3rd Party Libraries} \label{sec:3rdpkgs}
+
+Some of the packages listed in appendix \ref{sec:tomove} are already available in Conda public channels or \textit{PyPi} (see following \ref{sec:pypi}).
+They are now included in the Science Pipelines stack as EUPS packages since when they were initially needed, they were not available in any Conda channels.
+
+Some other packages require to be updated more often, therefore they have been included as EUPS packages, since the Conda environment is not sufficiently dynamic.
+
+Packages that requires customization, need to remain part of the stack, at least in a first phase.
+
+A small team shall be defined in DM to maintain the environment and the needed Conda recipes.
+
+
+\subsubsection{Conda Packages Managed by DM} \label{sec:3rddmpkgs}
+
+DM shall use the packages from the Conda public channels, if available.
+
+DM shall maintain a Conda recipes for all those packages that are not available in Conda / PyPi, or that requires to be rebuild with a specific flag. This is the case of \textit{boost}.
+These packages shall have a specific naming convention, so it will be clear they have been \textit{re-packaged} by LSST DM.
+
+A DM maintained Conda package shall be dismissed, not maintained anymore, as soon as the corresponding package available in the public Conda channel can be used in the Science Pipelines build.
+
+
+\subsubsection{PyPi Packages} \label{sec:pypi}
+
+The conda environment definition files in \textit{scipipe\_conda\_env} can accept also pip package. 
+
+This implies that packages available in \textit{PyPi} package repository and that can be installed in the environment. 
+However, if available, the preference shall be given to the Conda package.
+
+In case a pip package will bring into the environment too many PyPi dependencies, DM can decide to maintain a Conda recipe for it.
+
+
+\subsection{Change Control}
+
+Since maintaining conda recipes may require a considerable amount of effort, the DMCCB shall monitor when new packages are added to the environment or are used in the codebase.
+
+Changes to the environment shall be proposed using RFC Jira issues.
+
+Third party packages shall also be censed and documented together with the DM product tree. 
+Each package shall have an internal reference person.
```

Comment:
"Responsible" instead of "reference"?

Reply:
"responsible" mean responsibilities... and I don't see people be very happy to assume responsibilities. Using the word "reference" do not imply responsibilities.

If DM Arch, or DMCCB is willing to enforce an internal responsible on each 3rd party package, I will be very happy, however, having a reference person for the time been should be sufficient.